packages/server/migrations/20240308203616_drop_grants_raw_body_column.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * The text column `raw_body` was replaced with a jsonb column `raw_body_json` in a previous migration.
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.up = function (knex) {
+    return knex.schema.table('grants', (table) => {
+        table.dropColumn('raw_body');
+    });
+};
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+exports.down = function (knex) {
+    return knex.schema.table('grants', (table) => {
+        table.text('raw_body');
+    });
```

Comment:
What do you think of repopulating just in case?
```suggestion
    return knex.schema.table('grants', (table) => {
        table.text('raw_body');
    }).raw(`
        UPDATE grants 
          SET raw_body = raw_body_json::text 
        WHERE raw_body_json IS NOT NULL;
    `);
```

Reply:
Makes sense, changed.